site/content/docs/4.5/components/carousel.md

```diff
@@ -208,6 +208,34 @@ Add `data-interval=""` to a `.carousel-item` to change the amount of time to del
 </div>
 {{< /example >}}
 
+### Suppressing touch swiping
+
+By default, carousels support swiping left/right on touchscreen devices to move between slides. This can be explicitly suppressed using the `data-touch` attribute. The example below also does not include the `data-ride`, and has `data-interval="false"`, meaning that it won't autoplay.
```

Comment:
```suggestion
Carousels support swiping left/right on touchscreen devices to move between slides. This can be disabled using the `data-touch` attribute. The example below also does not include the `data-ride` and has `data-interval="false"` so it doesn't autoplay.
```

Reply:
addressed with separate commit, as i also noticed that this could do with the extra word "attribute" after `data-ride`